src/erlcloud_cloudformation.erl

```diff
@@ -0,0 +1,626 @@
+
+-spec get_template(string(), aws_config()) ->
+    {ok, cloudformation_list()} | {error, error_reason()}.
+get_template(StackName, Config = #aws_config{}) ->
+
+    case cloudformation_request(Config, "GetTemplate",
+            [{"StackName", StackName}]) of
+        {ok, XmlNode} ->
+            extract_template_response(XmlNode);
+        {error, Error} ->
+            {error, Error}
+    end.
+
+-spec get_template_summary(params(), string()) ->
+    {ok, cloudformation_list()} | {error, error_reason()}.
+get_template_summary(Params, StackName) ->
+    get_template_summary(Params, StackName, default_config).
+
+-spec get_template_summary(params(), string(), aws_config()) ->
+    {ok, cloudformation_list()} | {error, error_reason()}.
+get_template_summary(Params, StackName, Config = #aws_config{}) ->
+
+    FullParams = [{"StackName", StackName}
+        | lists:map(
+            fun(T) ->
+                convert_param(T)
+            end, Params)],
+
+    case cloudformation_request(Config, "GetTemplateSummary", FullParams) of
+        {ok, XmlNodes} ->
+            extract_template_summary_response(XmlNodes);
+        {error, Error} ->
+            {error, Error}
+    end.
+
+-spec describe_account_limits_all() ->
+    {ok, cloudformation_list()} | {error, error_reason()}.
+describe_account_limits_all() ->
+    describe_account_limits_all(default_config).
+
+-spec describe_account_limits_all(aws_config()) ->
+    {ok, cloudformation_list()} | {error, error_reason()}.
+describe_account_limits_all(Config = #aws_config{}) ->
+    list_all(fun describe_account_limits/2, [], Config, []).
+
+-spec describe_account_limits(aws_config(), string()) ->
+    {ok, cloudformation_list()} | {error, error_reason()}.
+describe_account_limits(Params, Config = #aws_config{}) ->
+
+    RequestParams = lists:map(fun(T) -> convert_param(T) end, Params),
+
+    case cloudformation_request(Config, "DescribeAccountLimits",
+            RequestParams) of
+        {ok, XmlNodes} ->
+            NextToken = erlcloud_xml:get_text(
+                "/DescribeAccountLimitsResponse/DescribeAccountLimitsResult/"
+                "NextToken",
+                XmlNodes,
+                undefined),
+
+            AccountLimits = extract_accout_limits_response(xmerl_xpath:string(
+                "/DescribeAccountLimitsResponse",
+                XmlNodes)),
+
+            {ok, AccountLimits, NextToken};
+        {error, Error} ->
+            {error, Error}
+    end.
+
+%%==============================================================================
+%% Internal functions
+%%==============================================================================
+
+convert_param({Key, Value}) ->
+    case Key of
+        stack_name -> {"StackName", Value};
+        stack_id -> {"StackName", Value};
+        logical_resource_id -> {"LogicalResourceId", Value};
+        action -> {"Action ", Value};
+        version -> {"Version", Value};
```

Comment:
do we allow to specify Version in other modules? 
if no, let's also keep it hidden there as well